typescript/ws-protocol/README.md

```diff
@@ -1,13 +1,11 @@
-# Foxglove Studio WebSocket protocol
+# Foxglove WebSocket server and client
 
-This package provides server and client implementations of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol) supports any data encoding, as long as it is supported by both client and server.
```

Comment:
I think we should keep the "This package provides…" portion since this might be the first landing page some people see (since it is the package readme on npm). Whatever decisions we make for the python package readme should be mirrored here or vice versa